packages/pubsub/lib/event.ts

```diff
@@ -0,0 +1,28 @@
+import type { DBUser, DBTeam, BillingMetric } from '@nangohq/types';
+
+type JsonValue = string | number | boolean | null | JsonValue[] | { [key: string]: JsonValue } | Record<string, any>;
```

Comment:
`import { JSONValue } from type-fest` doesn't work?

Reply:
no, our types like BillingMetric, DBUser, DBTeam are missing an Index signature for type `string`. Also one line to avoid importing an entire library ;)